crates/orchestrator/src/database/mongodb/mod.rs

```diff
@@ -83,28 +83,30 @@ impl Database for MongoDb {
         Ok(self.get_job_collection().find_one(filter, None).await?)
     }
 
-    async fn update_job_status(&self, job: &JobItem, new_status: JobStatus) -> Result<()> {
+    async fn update_job(
```

Comment:
by any chance is there a way to update all fields automatically? this seems that it can break silently tomorrow if I add a new column to jobs

Reply:
Done